### PR TITLE
docs(*) introduce guides

### PR DESCRIPTION
**Pull request by jakubdyszkiewicz**
### Summary

Here is an idea. I see that sometimes we lack formal process or we have what is called [tribal knowledge](https://en.wikipedia.org/wiki/Tribal_knowledge)

I'd like to have a place that we can share tips and write some guides that can help develop the project.

Automation is always better than manual steps but sometimes we cannot avoid it.

More examples:
* Refresh RELEASE.md and move it here
* Create notes how to debug AKS/EKS tests

### CODE REVIEW

**Review by nickolaev — APPROVED** [review]
Great idea. Totally support it.

**Review by lahabana — APPROVED** [review]
Very cool! Especially the e2e tests part.

File: docs/guides/e2e-test-tips.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# E2E tests tips
+
+E2E tests are less stable and slower than unit tests therefore it's useful to know some tricks
+
+## Faster environment setup
+
+When e2e tests are executed here are the steps
+
+1) Build containers
+2) Start Kubernetes Cluster 2
```
Comment (parkanzky) [inline docs/guides/e2e-test-tips.md:10]: Are they both Cluster 2, or should this be Cluster 1 ?

**Comment by parkanzky** [comment]
This is extremely helpful. Thanks so much for putting this together.

File: docs/guides/e2e-test-tips.md (hunk 1 above)
Comment (jakubdyszkiewicz) [reply docs/guides/e2e-test-tips.md:10]: should be cluster1

**Pull request merged**